.github/workflows/image.yml

```diff
@@ -67,18 +64,27 @@ jobs:
                         type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', 'main') }}
 
             -   name: Prepare build
+                id: prepare
                 run: |
+                    export BUILD_DATE=$(date -Iseconds)
+                    export FLAVOUR=${{ matrix.flavour.name }}
+                    
                     cat build/Dockerfile.prefix > Dockerfile
                     cat "flavour/${{ matrix.flavour.name }}/Dockerfile.flavour" >> Dockerfile
                     
                     # Download mo
                     curl -sSL https://raw.githubusercontent.com/tests-always-included/mo/master/mo -o mo
                     chmod +x mo
                     
-                    cat build/Dockerfile.suffix.mo | BUILD_DATE=$(date -Iseconds) FLAVOUR=${{ matrix.flavour.name }} ./mo > build/Dockerfile.suffix
-                    
+                    cat build/Dockerfile.suffix.mo | ./mo > build/Dockerfile.suffix
                     cat build/Dockerfile.suffix >> Dockerfile
                     
+                    {
+                        echo 'labels<<EOF'
+                        cat build/labels.txt.mo | ./mo | sed -re "s/^/                    /gm" # Add whitespace to insert it as annotations
```

Comment:
Why so complicated? Can't the whitespace be added in the source file (here labels.txt.mo) already?

Reply:
I thought I'd do it here because in this file I know the required whitespace and I could comment why it is added...

Reply:
Okay, a bit complex though, but fine with me.